src/main/java/com/idrsolutions/microservice/FormVuServlet.java

```diff
@@ -193,6 +195,7 @@ protected boolean validateRequest(final HttpServletRequest request, final HttpSe
         settingsValidator.validateBoolean("org.jpedal.pdf2html.inlineJavaScriptAndCSS", false);
         settingsValidator.validateBoolean("org.jpedal.pdf2html.noCheckboxOrRadioButtonImages", false);
         settingsValidator.validateString("org.jpedal.pdf2html.submitUrl", "[-a-zA-Z0-9@:%._\\+~#=]{1,256}([-a-zA-Z0-9()@:%_\\+.~#?&//=]*)", false);
+        settingsValidator.validateBoolean("experimentalFormVuAPI", false);
```

Comment:
```suggestion
```
Adding this setting is no longer required, it has been superseded by https://github.com/idrsolutions/formvu-microservice-example/commit/643c34626b998d46e737cd58ea4cc758d102f1fd which is now live.